src/ui/units/workbooks/components/WorkbookActions/WorkbookActions.tsx

```diff
@@ -91,12 +97,7 @@ export const WorkbookActions: React.FC<Props> = ({workbook, refreshWorkbookInfo}
                     }),
                 );
             },
-            text: (
-                <React.Fragment>
-                    <Icon data={ArrowRight} className={classNameIconAction} />
-                    {i18n('action_move')}
-                </React.Fragment>
-            ),
+            text: <DropdownAction icon={ArrowRight} text={i18n('action_move')} />,
         });
 
         if (workbook.permissions.copy) {
```

Comment:
This is an old change, but why did we put two nested `if` here?
```ts
if (workbook.permissions.move) {
    if (workbook.permissions.copy) {
        ...
    }
}
```

Reply:
probably here unnesessary put two nested if, fixed it